docs/data-analysis.md

```diff
@@ -11,7 +11,7 @@ The following helpers will help you loading the data into Pandas DataFrames, and
 Debugging a strategy (are there no buy signals, ...) can be very time-consuming.
 FreqTrade tries to help you by exposing a few helper-functions, which can be very handy.
 
-I recommend using Juptyer Notebooks for this analysis, since it offers a dynamic way to rerun certain parts.
+It's recommendet using Juptyer Notebooks for analysis, since it offers a dynamic way to rerun certain parts of the code.
```

Comment:
recommended

Reply:
i will stop ~~reccomending~~ ~~recommenting~~ recommending stuff in the documentation ... 
it's not that i wouldn't know that i fail this word each time ...

Reply:
yeah, each time 🤣 

the rest is okay, you can merge it